eng/Signing.props

```diff
@@ -8,128 +13,155 @@
+        <ItemsToSign Include="$(BaseOutputRootPath)corehost/**/ijwhost.dll" />
+        <ItemsToSign Include="$(BaseOutputRootPath)corehost/**/nethost.dll" />
+
+        <!-- Sign managed libraries in installer subset. -->
+        <ItemsToSign Include="$(ArtifactsBinDir)Microsoft.NET.HostModel/**/*.dll" />
+      </ItemGroup>
+
+      <!-- Sign ready-to-run binaries after crossgen is applied. -->
+      <ItemGroup Condition="'$(SignR2RBinaries)' == 'true'">
+        <ItemsToSign Include="$(CrossGenRootPath)**/*.dll" />
+      </ItemGroup>
+
+      <ItemGroup Condition="'$(SignMsiFiles)' == 'true'">
+        <ItemsToSign Include="$(ArtifactsPackagesDir)**/*.msi" />
+        <ItemsToSign Include="$(ArtifactsPackagesDir)**/*.cab" />
+      </ItemGroup>
+
+      <ItemGroup Condition="'$(SignBurnEngineFiles)' == 'true'">
+        <ItemsToSign Include="@(BundleInstallerEngineArtifact)" />
+      </ItemGroup>
+
+      <ItemGroup Condition="'$(SignBurnBundleFiles)' == 'true'">
+        <!-- Sign the bundles, now that the engine is reattached. Avoid re-signing the engine. -->
+        <ItemsToSign
+          Include="@(BundleInstallerExeArtifact)"
+          Exclude="@(BundleInstallerEngineArtifact)" />
+        <!-- Note: wixstdba is internal to the engine bundle and does not get signed. -->
+      </ItemGroup>
+
+      <ItemGroup Condition="'$(SignFinalPackages)' == 'true'">
+        <DownloadedSymbolPackages Include="$(DownloadDirectory)**\*.symbols.nupkg" />
+        <ItemsToSign Include="$(DownloadDirectory)**\*.nupkg" Exclude="@(DownloadedSymbolPackages)" />
+
+        <!-- The cross OS diagnostics symbol packages need to be signed as they are the only packages
+        that have a specific version of assets that are only meant to be indexed in symbol servers.
+        Since only *symbols.nupkg get indexed, and installer doesn't produce these, we need to glob them for signing. -->
+        <ItemsToSign Include="$(DownloadDirectory)**\*CrossOsDiag*.symbols.nupkg" />
+
+        <ItemsToSign Include="$(DownloadDirectory)**\*.deb" />
+        <ItemsToSign Include="$(DownloadDirectory)**\*.rpm" />
+      </ItemGroup>
+    </When>
+    
+    <!-- When doing post build signing, we sign all artifacts we would push.
+         Symbol packages are included, but get no signature. -->
+    <When Condition="'$(PostBuildSign)' == 'true'">
+      <ItemGroup>
+        <ItemsToSignWithPaths Include="$(DownloadDirectory)**" />
+        <!-- Exclude wixpdb files from the signing list. These are generated into the
+             shipping/nonshipping directories but should be generated into the obj dirs.
+             https://github.com/dotnet/arcade/issues/6187. -->
+        <ItemsToSignWithPaths Remove="@(DownloadedWixPdbFile)" />
+        <ItemsToSignWithoutPaths Include="@(ItemsToSignWithPaths->'%(Filename)%(Extension)')" />
+        <ItemsToSign Include="@(ItemsToSignWithoutPaths->Distinct())" />
```

Comment:
Is out of build signing using file name only in ItemsToSign? Makes sense, but that's not how things work right now. Just wanted to make sure.

Reply:
Yes, because out of build signing doesn't have the same relative paths that the repo does.